com.unity.ml-agents/Runtime/Actuators/ActionSpec.cs

```diff
@@ -94,5 +93,39 @@ internal void CheckAllContinuousOrDiscrete()
                 );
             }
         }
+
+        public static ActionSpec Combine(ActionSpec[] specs)
```

Comment:
```suggestion
        public static ActionSpec Combine(ILIst<ActionSpec> specs)
```
and use `for` instead of `foreach`?